### PR TITLE
feat(route): chlinlearn daily bolg

### PR DESCRIPTION
**Pull request by huyyi**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/chlinlearn/daily-blog
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/chlinlearn/namespcae.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import type { Namespace } from '@/types';
+
+export const namespace: Namespace = {
+    name: '值得一读技术博客 ',
```
Comment (TonyRL) [inline lib/routes/chlinlearn/namespcae.ts:4]:
```suggestion
    name: '值得一读技术博客',
```

File: lib/routes/chlinlearn/daily-blog.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            link: item.url,
+            author: item.author,
+            img: item.icon,
+            pubDate: timezone(parseDate(item.publishTime), +8),
+        }));
+        return {
+            // 源标题
+            title: `值得一读技术博客`,
+            // 源链接
+            link: `https://daily-blog.chlinlearn.top/blogs/1`,
```
Comment (TonyRL) [inline lib/routes/chlinlearn/daily-blog.ts:43]:
```suggestion
            title: '值得一读技术博客',
            // 源链接
            link: 'https://daily-blog.chlinlearn.top/blogs/1',
```

File: lib/routes/chlinlearn/daily-blog.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch'; // 统一使用的请求库
+import { parseDate } from '@/utils/parse-date'; // 解析日期的工具函数
+import timezone from '@/utils/timezone';
+
+export const route: Route = {
+    path: '/daily-blog',
+    name: '值得一读技术博客',
```
Comment (TonyRL) [inline lib/routes/chlinlearn/daily-blog.ts:8]: name should not be the same as namespace's name

Comment (huyyi) [reply lib/routes/chlinlearn/daily-blog.ts:8]: updated the namespace name to "chlinlearn 的技术博客"

File: lib/routes/chlinlearn/namespcae.ts [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import type { Namespace } from '@/types';
+
+export const namespace: Namespace = {
+    name: 'chlinlearn的技术博客',
+    url: 'https://daily-blog.chlinlearn.top/',
```
Comment (TonyRL) [inline lib/routes/chlinlearn/namespcae.ts:5]:
```suggestion
    url: 'daily-blog.chlinlearn.top',
```

**Pull request merged**